Common/Deployment/PrepareIoTSample.ps1

```diff
@@ -27,15 +27,15 @@ switch($azureEnvironmentName)
         $global:websiteSuffix = "azurewebsites.net"
         $global:locations = @("East US", "North Europe", "East Asia", "West US", "West Europe", "Southeast Asia", "Japan East", "Japan West", "Australia East", "Australia Southeast")
     }
-    "AzureGermanyCloud" {
-        if ((Get-AzureEnvironment AzureGermanyCloud) -eq $null)
+    "AzureGermanCloud" {
```

Comment:
this is weird. I had fixed this in master and AzureGermanCloud is the right name: https://github.com/Azure/azure-iot-remote-monitoring/pull/222/files
But somewhat this part got reverted back :/